src/main/scala/outwatch/dom/VDomModifier.scala

```diff
@@ -51,25 +51,21 @@ final case object EmptyVDomModifier extends VDomModifier
 
 
 object VDomModifier {
-  final implicit class StringNode(string: String) extends VNode {
-    val asProxy = VNodeProxy.fromString(string)
+  final implicit class StringNode(val string: String) extends AnyVal with VNode {
+    def asProxy = VNodeProxy.fromString(string)
   }
 
   implicit def OptionIsEmptyModifier(opt: Option[VDomModifier]): VDomModifier = opt getOrElse EmptyVDomModifier
 
-  final case class VTree(nodeType: String,
-                   children: Seq[VNode],
-                   attributeObject: DataObject
-                  ) extends VNode {
+  final case class VTree(nodeType: String, modifiers: Vector[VDomModifier]) extends VNode {
```

Comment:
Why make this a vector? I think for small collections like this it makes more sense to have a `List`, no?

Reply:
`Vector` has better complexity for concatenation and `lastOption`. But for these short lists we need a benchmark on the javascript runtime. Maybe arrays are the fastest. Arrays would also have the advantage that we could directly store the passed arguments without conversion.